helm/orca/config/alerts-mapping.yaml

```diff
@@ -232,6 +648,49 @@ falco:
         properties:
           name: k8s.pod.name
           namespace: k8s.ns.name
+    - name: "Create HostNetwork Pod"
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: ka.resp.name
+          namespace: ka.target.namespace
+    - name: "Create/Modify Configmap With Private Credentials"
+      source_mapping:
+        origin: kubernetes
+        kind: config_map
+        properties:
+          name: ka.req.configmap.name
+    - name: "Attach/Exec Pod"
+      source_mapping:
+        origin: kubernetes
+        kind: pod
+        properties:
+          name: ka.target.name
+          namespace: ka.target.namespace
+    - name: "Create Disallowed Namespace"
+      source_mapping:
+        origin: kubernetes
+        kind: namespace
+        properties:
+          name: ka.target.name
+    - name: "Ingress Object without TLS Certificate Created"
+      source_mapping:
+        origin: kubernetes
+        kind: ingress
+        properties:
+          name: ka.target.name
+          namespace: ka.target.namespace
+    - name: "Untrusted Node Successfully Joined the Cluster"
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: "Untrusted Node Unsuccessfully Tried to Join the Cluster"
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
```

Comment:
At first, I had the same mapping intuition, however, if you extract the essence of these two alerts, they come down to a warning: `The node is untrusted`. The fact that a node is *untrusted* is the key information, not the fact that the node joined a cluster - it's a regular behaviour, we already record these events. I think it would be valuable for an operator to highlight in the graph the nodes that have been identified as *untrusted*.

Is it possible to map the above alerts to node entities?

Reply:
Yes, It's possible. I just wasn't sure if the node would be placed in the graph in case of "trying to join unsuccessfully", so I mapped the alert to the cluster kind. I'm gonna change it! ;)